### ISSUE TITLE
dasher-hue stopped working

### ISSUE THREAD

**Issue by ac2323** [Ricardo1980/dasher-hue#2]
For no apparent reason, dasher-hue stopped working. Nothing has changed in my environment. I noticed that the dash button presses are no longer being detected...I tried reinstalling without success. Any ideas?

libpcap-dev v1.8.1-6ubuntu1
npm v5.8.0+ds-2

**Comment by Ricardo1980** [comment]
A couple of ideas:
Use script/find_button from 
https://github.com/maddox/dasher/blob/master/README.md
to check if your network is receiving any signal. 

Check also the IP address, maybe it is different now if you restarted everything. 

Worst case scenario, test this one:
https://github.com/Nekmo/amazon-dash

In any case, my app is just a wrapper, working perfectly in my home since I uploaded it. I bet is the ip address or your network is not receiving any signal.

**Comment by ac2323** [comment]
Whenever I press the button, I see it come up on my router. For some reason dasher-hue isn't detecting the press. I tried script/find_button but nothing is detected. I do get "Possible dash hardware address detected: [MAC] Manufacturer: unknown Protocol: udp" which i've never seen before and does not correspond to a valid MAC on my network.


What IP address am I checking? The only specified in the config is the bridge IP which is statically set on my router.

Tried Nekmo's amazon-dash and the result is the same.
